lib/Ix/Util.pm

```diff
@@ -58,6 +59,14 @@ sub differ ($x, $y) {
   return 1 if defined $x xor defined $y;
   return unless defined $x;
 
+  if (JSON::MaybeXS::is_bool($x)) {
```

Comment:
From discussion: probably instead we should be inflating bools to objects when they come out of the db.